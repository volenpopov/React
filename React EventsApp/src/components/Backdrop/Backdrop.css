.backdrop {    
    width: 100%;
    height: 100vh;
    background: rgba(0, 0, 0, 0.55);
    position: fixed;
    top: 0;
    left: 0;
}