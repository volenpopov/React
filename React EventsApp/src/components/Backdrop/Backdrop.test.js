import React from 'react';

import { shallow } from 'enzyme';

import Backdrop from "./Backdrop";

describe("<Backdrop />", () => {
    it("expect to match snapshot", () => {
        expect(shallow(<Backdrop/>)).toMatchSnapshot();
    });
});
