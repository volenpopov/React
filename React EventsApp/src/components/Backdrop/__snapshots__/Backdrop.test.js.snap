// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Backdrop /> expect to match snapshot 1`] = `ShallowWrapper {}`;
